controllers/Performances.js

```diff
@@ -3,6 +3,21 @@ const Performance = new models.Performance();
 const Result = new models.Result();
 
 function PerformanceController() {
+  this.create = (req, res) => {
+    const { performanceName, performanceDate, openAt, closeAt, current } = req.body;
+
+    Performance.create(performanceName, performanceDate, openAt, closeAt, current)
+    .then(({ id }) => {
+      // TODO: create cron job
+      console.log('Performance id', id);
+      res.redirect(`/${req.user.nameNumber}`);
```

Comment:
Right now, the fetch is just displaying a banner message, so this redirect doesn't really do anything. It would make more sends to just send a success message or something like that